telco-core/configuration/reference-crs-kube-compare/required/performance/PerformanceProfile.yaml

```diff
@@ -6,8 +6,14 @@ metadata:
   name: {{ .metadata.name }}
   annotations:
     # Some pods want the kernel stack to ignore IPv6 router Advertisement.
-    kubeletconfig.experimental: |
-      {"allowedUnsafeSysctls":["net.ipv6.conf.all.accept_ra"]}
+    {{ $allowedSysctls := (list
+        "net.ipv6.conf.IFNAME.accept_ra"
+        "net.ipv6.conf.IFNAME.disable_ipv6"
+        "net.ipv4.conf.IFNAME.disable_policy"
+        "net.ipv6.conf.IFNAME.disable_policy"
+        "net.ipv4.conf.IFNAME.rp_filter"
+        "net.ipv6.conf.IFNAME.hop_limit") }}
```

Comment:
For this list we will need to expand the IFNAME to "all" and "default" (ie two entries per). We can only use the IFNAME syntax when used within the csi